components/images/images-en.html

```diff
@@ -0,0 +1,249 @@
+---
+{
+	"altLangPage": "images-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" },
+		{ "title": "Provisional functionality", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-en.html" }
+	],
+	"dateModified": "2023-07-12",
+	"description": "Images WET-BOEW Component",
+	"language": "en",
+	"title": "Images"
+}
+---
+<span class="wb-prettify all-pre hide"></span>
+
+<h2>Working examples</h2>
+
+<h3>Shapes</h3>
+<p>Use to easily modify the shape of an <code>&lt;img&gt;</code> (image) without the use of editing software.</p>
+
+<h4>Appearance</h4>
```

Comment:
having the heading "Appearance" don't add value in the heading hierarchy on this page as its content can fade directly under the heading "Shapes". And do the same for the other section

Reply:
Fixed